students/ghassan/lesson03/slicing.py

```diff
@@ -0,0 +1,44 @@
+#!/usr/bin/env python3
+
+
+def first_last(data):
+    new_list = data[:]
+    new_list[0] = data[-1]
+    new_list[-1] = data[0]
+    return new_list
+
+
+def every_other(data):
+    new_list = data[::2]
+    return new_list
+
+
+def fourth_fourth(data):
+    new_list = data[0:4] + data[-4:]
+    return new_list
+
+
+def reversed(data):
```

Comment:
be careful here.. you are overwriting built-in function `reversed` (notice github highlight it different color, so should your editor)